scripts/r.fillnulls/testsuite/test_r_fillnulls.py

```diff
@@ -18,46 +24,87 @@ class TestRFillNulls(TestCase):
     expression = "elevation_filt = if(elevation > 130, null(), elevation)"
     mapNameCalc = "elevation_filt"
     mapComplete = "elevation_complete"
+    mapInt = "elevation_int"
     values = "null_cells=0"
 
     def setUp(self):
         """Create maps in a small region."""
         self.use_temp_region()
         self.runModule("g.region", res=200, raster=self.mapName, flags="ap")
-        run_command("r.mapcalc", expression=self.expression)
+        run_command("r.mapcalc", expression=self.expression, overwrite=True)
 
     def tearDown(self):
         """Remove temporary region"""
         self.runModule(
             "g.remove",
             flags="f",
             type="raster",
-            name=(self.mapNameCalc, self.mapComplete),
+            name=(self.mapNameCalc, self.mapComplete, self.mapInt),
         )
         self.del_temp_region()
 
-    def test_rst(self):
+    def test_basic(self):
+        """Test using default RST interpolation"""
         module = SimpleModule(
             self.module,
             input=self.mapNameCalc,
             output=self.mapComplete,
             segmax=1200,
-            npmin=100,
+            npmin=50,
             tension=150,
+            overwrite=True,
         )
         self.assertModule(module)
         self.assertRasterFitsUnivar(raster=self.mapComplete, reference=self.values)
 
-    def test_bspline(self):
+    def test_bicubic(self):
+        """Test using bicubic interpolation"""
         module = SimpleModule(
             self.module,
             input=self.mapNameCalc,
             output=self.mapComplete,
             method="bicubic",
+            overwrite=True,
         )
         self.assertModule(module)
         self.assertRasterFitsUnivar(raster=self.mapComplete, reference=self.values)
 
+    def test_bilinear(self):
+        """Test using bilinear interpolation"""
+        module = SimpleModule(
+            self.module,
+            input=self.mapNameCalc,
+            output=self.mapComplete,
+            method="bilinear",
+            lambda_="0.02",
+            overwrite=True,
+        )
+        self.assertModule(module)
+        self.assertRasterFitsUnivar(raster=self.mapComplete, reference=self.values)
```

Comment:
Instead of just checking there are no null cells, please also include other stats like mean, min, max